### PR TITLE
Feature/phone email extras

### LINKED ISSUE #260: Add content under phone and email
o   (under email) This email address will be used to send you a confirmation and instructions for this application. The phone number will be used to contact you if additional details are required.
o   (A checkbox needs to be added under phone number with the text) Are you happy for the email address and/or phone number to be used for other Council communications?

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?
Added checkbox for textfields
renamed TextField component

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] tested in Chrome
- [ ] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

closes #260

### CODE REVIEW

File: src/styles/scss/App.scss [hunk 1]
```diff
@@ -538,3 +538,72 @@ a {
...
+    left: -9999px;
+    &[type=checkbox]:checked + span:after {
+      content: "";
+      position: absolute;
+      left: 12px;
+      top: 8px;
+      width: 10px;
+      height: 20px;
+      border: solid $deep-grey1;
+      border-width: 0 4px 4px 0;
```
[the remark this answers is not in the document]
Comment (dlouise64) [reply src/styles/scss/App.scss]: this is fine.

**Comment by dlouise64** [comment]
@Br3nda Ready for review

**Comment by dlouise64** [comment]
closes #260

**Comment by dlouise64** [comment]
screen shot 2018-05-23 at 3 41 44 pm

**Comment by dlouise64** [comment]
@Br3nda just double checked locally and can see its come through see image

**Comment by Br3nda** [comment]
hmmm.. dunno why i didn't find it.
all aproved now

**Pull request merged**